backend/tracim_backend/error_code.py

```diff
@@ -0,0 +1,54 @@
+# Error code format : [TYPE:2c]-[NUMBER:2c]
+# TYPE:
+# NF=Not found error
+# RI=Right error
+# VA=validation post-marshallow
+# CO=Conflict error
+# AT=Auth error
+# MV = marshmallow schema validation error
+
```

Comment:
- I suggest to rationalize error types. Too much families = risk of misuse.
- I prefer to keep error codes being integers.
- By the way, the format you suggest is limited to 99 error codes per family and you already use 30 on a family - it's not enough.

Here are the families and codes I suggest for now: 

- define error codes as integers defined by 4 digits.
- the error are grouped like this:
   - `1xxx`: not found error
   - `2xxx`: validation error
   - `3xxx`: conflict error
   - `4xxx`: authentication and authorization
   - `9xxx`: core errors (family of error code reserved for unclassable errors or very low level errors)